examples/meteringlab/c1/cluster.yaml

```diff
@@ -0,0 +1,202 @@
+# Namespace where the Scylla Cluster will be created
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: scylla
+
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: xfs-nvme-249g-iotune
+  namespace: scylla
+data:
+  io_properties.yaml: |
+    disks:
+    - mountpoint: /var/lib/scylla
+      read_iops: 385400
+      read_bandwidth: 3087631104
+      write_iops: 240628
+      write_bandwidth: 2289281280
+
+---
+
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: NodeConfig
+metadata:
+  name: cluster
+spec:
+  localDiskSetup:
+    filesystems:
+    - device: nvmes
+      type: xfs
+    mounts:
+    - device: nvmes
+      mountPoint: /mnt/persistent-volumes
+      unsupportedOptions:
+      - prjquota
+    raids:
+    - name: nvmes
+      type: RAID0
+      RAID0:
+        devices:
+          modelRegex: Amazon EC2 NVMe Instance Storage
+          nameRegex: ^/dev/nvme\d+n\d+$
+  placement:
+    nodeSelector:
+      scylla.scylladb.com/node-type: scylla
```

Comment:
nodeSelector doesn't match the nodegroups where it should be deployed nor it doesn't have required toleration for the taint specified on scylla node group.